packages/e2e/package.json

```diff
@@ -31,5 +32,10 @@
     "selenium-server": "^3.141.59",
     "ts-node": "^9.0.0",
     "typescript": "^4.0.3"
+  },
+  "dependencies": {
```

Comment:
Did you add them to dependencies deliberately? Consider moving them to devDependencies.

Reply:
Still stands.